docs/commands/ft.create.md

```diff
@@ -1,238 +1,270 @@
-Creates an index with the given spec.
+---
+syntax: 
+---
 
+Create an index with the given specification
 
-{{% alert title="Note on attribute number limits" color="warning" %}}
-  RediSearch supports up to 1024 attributes per schema, out of which at most 128 can be TEXT attributes.
-  On 32 bit builds, at most 64 attributes can be TEXT attributes.
-  Note that the more attributes you have, the larger your index will be, as each additional 8 attributes require one extra byte per index record to encode.
-  You can always use the `NOFIELDS` option and not encode attribute information into the index, for saving space, if you do not need filtering by text attributes. This will still allow filtering by numeric and geo attributes.
-{{% /alert %}}  
+## Syntax
 
-{{% alert title="Note on running in clustered databases" color="info" %}}
-  When having several indices in a clustered database, you need to make sure the documents you want to index reside on the same shard as the index. You can achieve this by having your documents tagged by the index name.
-{{% /alert %}}  
+{{< highlight bash >}}
+FT.CREATE index [ON HASH | JSON] [ PREFIX count prefix [prefix ...]] [LANGUAGE default_lang] [LANGUAGE_FIELD lang_attribute] [SCORE default_score] [SCORE_FIELD score_attribute] [PAYLOAD_FIELD payload_attribute] [MAXTEXTFIELDS] [TEMPORARY seconds] [NOOFFSETS] [NOHL] [NOFIELDS] [NOFREQS] [STOPWORDS count stopword [stopword ...]] SCHEMA field_name [AS alias] TEXT | TAG | NUMERIC | GEO | VECTOR [ SORTABLE [UNF]] [NOINDEX] [ field_name [AS alias] TEXT | TAG | NUMERIC | GEO | VECTOR [ SORTABLE [UNF]] [NOINDEX] ...]
```

Comment:
Again, we do not want to manually repeat auto-generated content (in any command)
If we want to change the formatting/style, should probably be done in the generator.

Reply:
The syntax is not repeated, just copied and pasted to avoid long command titles. This format is not uncommon in documentation in general. It's nice to have it autogenerated, but, because it won't change that often, I don't think it's unreasonable to have it in a font and style that is easier to read.